src/modules/account/store/selectors.js

```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
 export const selectCurrentAccount = state => state.account.current;
```

Comment:
What do you want to do with `selectActiveTokenAccount` then?

Reply:
its gonna be remove! 
we have the new selector for it, also we have the useCurrentAcccount hook for the account
 by the way those change is not related to this PR!!